python/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ set(ignore
     gtsam::Pose3Vector
     gtsam::KeyVector
     gtsam::BinaryMeasurementsUnit3
+    gtsam::CameraSetCal3Bundler
+    gtsam::CameraSetCal3_S2
```

Comment:
Why are these here? `CameraSet` is a valid type rather than a typedef of an STL container. You should be able to access the class methods without specializing this class.

Reply:
@ProfFan should be able to explain. Is it not explained in the README? I asked @ProfFan to ask a README to explain what goes here.

Reply:
It is a STL container (and actually templated on the inner type as well), so this is required.

Reply:
Note this is not merely for accessibility:
- You do not ignore the class, then you have STL binding but needs one extra copy (`<stl.h>`)
- You ignore and `MAKE_OPAQUE`, then you have a fast binding without copy (`<stl_bind.h>`)

The automatic STL binding is nice, but I think it's always better to avoid overheads :)

Reply:
I see. This raises the question of whether code that accepts a CameraSet would accept a List or would it be expecting an object of type CameraSet. This is the weird thing about this scenario since CameraSet is not a vector alias, rather a vector subclass.